### PR TITLE
feat: removed the sock links

### PR DESCRIPTION
**Pull request by Anas12091101**
### What are the relevant tickets?
Fixes https://github.com/mitodl/hq/issues/3482

### Description (What does it do?)
Removes the sock links from the legacy studio

#### Before
Screenshot 2024-04-03 at 11 22 20 AM

#### After
Screenshot 2024-04-01 at 1 13 02 PM

### How can this be tested?
- Setup mitxonline-theme on your local edx-platform instance.
- Go to studio.
- You wont see any "Looking for help with studio?" drop down.

### CODE REVIEW

File: cms/templates/index.html
Comment (Anas12091101) [inline cms/templates/index.html:1]: `index.html` file is added to remove the sidebar message which was mentioning the sock links to get help from and since we removed the sock links, we don't need this message

Comment (arslanashraf7) [reply cms/templates/index.html:1]: I would have overridden the index.html as a last resort because this file is more prone to changes than other templates we override. I say this because index.html is the starting point; ideally, we'll have to keep it in sync with the upstream index.html updates. 

So, I have a question. How important is it to remove the `New to Studio` paragraph? @pdpinch 
image

**Review by arslanashraf7 — APPROVED** [review]
👍 The changes look good. However, I had a concern and left a comment, Please wait for Peter's response before you merge.

**Comment by arslanashraf7** [comment]
@Anas12091101 Based on https://github.com/mitodl/brand-mitxonline/pull/1#issuecomment-2041620002 and https://github.com/mitodl/mitxonline-theme/pull/42#pullrequestreview-1976173442. Since we don't want to hide the help dialog. We should hide the links with an empty sock.html file I think instead of index.html.

**Comment by Anas12091101** [comment]
I have removed the `index.html` file. Are we good to merge it @arslanashraf7 ?

**Comment by arslanashraf7** [comment]
> I have removed the `index.html` file. Are we good to merge it @arslanashraf7 ?

Yes

**Pull request merged**
